package.json

```diff
@@ -7,9 +7,10 @@
     "coverage": "neutrino test --coverage"
   },
   "devDependencies": {
-    "enzyme": "^2.8.2",
+    "@neutrinojs/jest": "^8.3.0",
```

Comment:
We should update the `yarn.lock` file as these changes. Could you add the changed file to your PR?